doc/webui/index.rst

```diff
@@ -0,0 +1,124 @@
+.. _webui:
+
+WebUI
+=========
+
+privacyIDEA comes with a web-based user interface which is used to manage and configure
+the privacyIDEA server. With the correct policies, it is also used a self-service portal
```

Comment:
Administrators can log in to configure the system and manage other users tokens.
Normal users can login and manage their own tokens.

Reply:
Please remove "with the correct policies".

Reply:
ok

Reply:
done